.gitignore

```diff
@@ -1,6 +1,7 @@
 *.gem
 *.rbc
 *.swp
+*.nix
```

Comment:
Rational XDE files? https://whatext.com/nix

Reply:
Nix language files from [NixOS](https://nixos.org) ;) I use nixpkgs + a per-project `shell.nix` on Darwin instead of Homebrew (and also on various non-NixOS Linux distributions). Together with `direnv` it makes a supercharged bundler/virtualenv/docker alternative.